core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/FixedRateSamplingTelemetryProcessor.java

```diff
@@ -41,14 +47,17 @@
 @BuiltInProcessor("FixedRateSamplingTelemetryProcessor")
 public final class FixedRateSamplingTelemetryProcessor implements TelemetryProcessor {
 
-    private final String dependencyTelemetryName = "Dependency";
-    private static final String eventTelemetryName = "Event";
-    private static final String exceptionTelemetryName = "Exception";
-    private static final String pageViewTelemetryName = "PageView";
-    private static final String requestTelemetryName = "Request";
-    private static final String traceTelemetryName = "Trace";
+    public static final double DEFAULT_SAMPLING_PERCENTAGE = 100.;
```

Comment:
trailing dot is syntactically correct? if so, there's nothing wrong here. I just didn't know that 😄

Reply:
Doesn't fail, but probably typo. I don't know either. I will make it 100.0 :)

Reply:
Probably for the best. It's an easy character to miss if you're skimming.

Reply:
#resolved